scripts/linters/python_linter.py

```diff
@@ -195,7 +195,7 @@ def _get_jobs_class_names_in_filepath(filepath, base_class_name):
         one_off_jobs_list = []
         validation_jobs_list = []
         for filepath in self.all_filepaths:
-            if filepath.endswith('prod_validation_jobs_one_off.py'):
+            if filepath.endswith('_validation_jobs_one_off.py'):
```

Comment:
Can you explain the reason for this change? Also, the jobs in `core/domain/interaction_validation_jobs_one_off.py` have `ProdValidationAuditOneOffJob` base?

Reply:
Reverted this actually - kind of got confused and forgot that these jobs have a different base and are not storage model validation. Thanks for pointing this out!